runtime/convertValues_test.go

```diff
@@ -4139,8 +4013,8 @@ func TestRuntimePublicKeyImport(t *testing.T) {
 
 	t.Run("Missing raw public key", func(t *testing.T) {
 		script := `
-            pub fun main(key: PublicKey): Bool {
-                return key.isValid
+            pub fun main(key: PublicKey): PublicKey {
+                return key
```

Comment:
I'm not sure if this change is sensible. Since there isn't a `PublicKey.isValid` anymore, I just did the minimum change to make the syntax work.